hazelcast/src/main/java/com/hazelcast/jet/core/ProcessorMetaSupplier.java

```diff
@@ -182,6 +182,20 @@ default boolean closeIsCooperative() {
     default void close(@Nullable Throwable error) throws Exception {
     }
 
+    /**
+     * Returns {@code true} if this instance is reusable, i.e. {@link #init}
+     * and {@link #close} can be called multiple times, possibly in parallel.
+     * <p>
+     * When a job is to be submitted, the job definition ({@link DAG} or
+     * {@link Pipeline}) is serialized. This serialization can be avoided
+     * for light jobs if the DAG is reusable.
+     *
+     * @since 5.3
+     */
+    default boolean isReusable() {
```

Comment:
We can also call it `isIdempotent()`, but the term has other meanings in different contexts.